src/main/java/bio/terra/service/snapshot/SnapshotService.java

```diff
@@ -444,9 +446,11 @@ private void conjureSnapshotTablesFromDatasetTables(Snapshot snapshot,
             List<Column> columnList = new ArrayList<>();
             List<SnapshotMapColumn> mapColumnList = new ArrayList<>();
 
-            // for each dataset column specified in the request, create a column in the snapshot with the same name
+            // for each dataset column specified in the request, create a column in the snapshot w the same name & array
             for (Column datasetColumn : datasetTable.getColumns()) {
-                Column snapshotColumn = new Column().name(datasetColumn.getName());
+                Column snapshotColumn = new Column()
+                    .name(datasetColumn.getName())
```

Comment:
Same, you'll want the datatype

Reply:
It might make sense to have some utility method like `createSnapshotColumnFromDatasetColumn`

Reply:
Instead of a utility method, I would make it a `Column` method and name is something like `toSnapshotColumn()`

Reply:
Fair enough. That works too.  In this case we don't have to worry about having to convert to a million different classes